### PR TITLE
✅(CI/CD) : Deployment and hosting workflows 

### LINKED ISSUE #287: Host website

### LINKED ISSUE #288: Configure AWS Deployment
Adjust inbound and outbound rules
setup EC2, S3, and elastic beanstalk environment

### LINKED ISSUE #289: Build size reduction script
Clean out unused icons after build

### LINKED ISSUE #290: Create CD workflows
Auto deploy app and API for hosting

### LINKED ISSUE #291: Adjust API endpoints and API calls
update endpoints to be open for CORS policy 
update API calls to match hosted service

### PR DESCRIPTION
**Pull request by priyo5**
Closes #287 
Closes #288 
Closes #289 
Closes #290 
Closes #291

### CODE REVIEW

File: tools/scripts/clean-build.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+            fromDir(filename, filter, callback);
+        }
+        else if (filter.test(filename)) {
+            callback(filename, files[i])
+        }
+    }
+}
+
+function justPrint(filePath, fileName){
+    console.log('-- found: ', fileName.split('.')[0]);
```
Comment (Ruben-Brits) [inline tools/scripts/clean-build.js:25]: do we need this console log?

**Review by Ruben-Brits — CHANGES_REQUESTED** [review]

File: tools/scripts/clean-build.js (hunk 1 above)
Comment (priyo5) [reply tools/scripts/clean-build.js:25]: It provides some good detail as to which assets are being removed

Comment (priyo5) [reply tools/scripts/clean-build.js:25]: In the event that something fails in the pipeline and it is this we have a clear indicator

Comment (Ruben-Brits) [reply tools/scripts/clean-build.js:25]: Just need to make sure this doesn't pop up in the browser console for production

**Review by Ruben-Brits — APPROVED** [review]

**Review by PxttytheThxtty — APPROVED** [review]

**Pull request merged**